src/bridge/command.rs

```diff
@@ -181,19 +239,20 @@ fn platform_which(bin: &str) -> Option<String> {
 }
 
 #[cfg(target_os = "macos")]
-fn nvim_cmd_impl(bin: String, mut args: Vec<String>) -> TokioCommand {
-    let shell = env::var("SHELL").unwrap_or_else(|_| "/bin/sh".to_string());
-    args.insert(0, bin);
-    let args = match shlex::try_join(args.iter().map(String::as_str)) {
-        Ok(args) => args,
-        Err(_) => panic!("Failed to join arguments"),
-    };
-    let mut cmd = TokioCommand::new(shell);
-    if env::var_os("TERM").is_none() {
-        cmd.arg("-l");
-    }
-    cmd.arg("-c");
-    cmd.arg(&args);
+pub fn is_tty() -> bool {
```

Comment:
Maybe I'm missing something dumb, but shouldn't this return true if stdout is a terminal? I think I'm missing something. Is it possible we could rename this function to something that makes that confusion less likely?

Reply:
Actually you are right. In cases like Finder/Dock/Neohub it should be considered as non tty, my confusion. So we should basically update the code to

```rust
#[cfg(target_os = "macos")]
pub fn is_tty() -> bool {
    std::io::stdout().is_terminal()
}
```

which teturns `true` if the descriptor/handle refers to a terminal/tty.

Regarding the function name, keeping it like it is, has the right term now.